.github/workflows/build-and-push.yml

```
@@ -0,0 +1,62 @@
+      - name: Docker login
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Set image tag (commit SHA)
+        id: vars
+        run: echo "tag=${GITHUB_SHA}" >> "$GITHUB_OUTPUT"
+
+      - name: Build and Push Docker image
+        run: |
```

actionlint (1.7.11)

[error] property "additional_tag" is not defined in object type {additional-tag: string}

(expression)